.github/workflows/ci.yml

```diff
@@ -269,3 +269,114 @@ jobs:
+      #
+      -
+        name: Export docker-image tag(s) for main and release-* git-branches
+        if: startsWith(github.ref, 'refs/heads/')
+        run: |
+          TAGS="${DOCKER_IMAGE_NAME}:${GITHUB_REF_NAME//main/latest}"
+
+          echo "DOCKER_IMAGE_TAGS=${TAGS}" | tee -a $GITHUB_ENV
+      #
+      # git-tag    : 11.5.1
+      # image-tags : typo3solr/ext-solr:11.5.1,typo3solr/ext-solr:11.5,typo3solr/ext-solr:11
+      #
+      # git-tag    : 11.2.1
+      # image-tags : typo3solr/ext-solr:11.2.1,typo3solr/ext-solr:11.2
+      #
+      # git-tag    : 10.0.5
+      # image-tags : typo3solr/ext-solr:10.0.5,typo3solr/ext-solr:10.0,typo3solr/ext-solr:10
+      #
+      -
+        name: Export docker-image tag(s) for release git-tags
+        if: ( startsWith(github.ref, 'refs/tags/') && !contains(github.ref, '-') )
+        run: |
+          # image-tag: typo3solr/ext-solr:11.5.1
+          TAGS="${DOCKER_IMAGE_NAME}:${GITHUB_REF_NAME}"
+          # image-tag: typo3solr/ext-solr:11.5
+          TAGS="${DOCKER_IMAGE_NAME}:${GITHUB_REF_NAME%.*}${TAGS+,$TAGS}"
+
+          VERSION_MAJOR_ONLY="${GITHUB_REF_NAME%%.*}"
+          VERSION_MAJOR_MINOR="${GITHUB_REF_NAME%.*}"
+          VERSION_LASTEST=$(git tag -l --sort -version:refname | grep -E "^${VERSION_MAJOR_ONLY}\\." | head -n1)
+
+          if [ "${VERSION_LASTEST%.*}" = "${VERSION_MAJOR_MINOR}" ]; then
+            # image-tag: typo3solr/ext-solr:11
+            TAGS="${DOCKER_IMAGE_NAME}:${VERSION_MAJOR_ONLY}${TAGS+,$TAGS}"
+          fi
+
+          echo "DOCKER_IMAGE_TAGS=${TAGS}" | tee -a $GITHUB_ENV
+      #
+      # git-tag    : 11.5.0-rc-3
+      # image-tag  : typo3solr/ext-solr:11.5.x-dev
+      #
+      -
+        name: Export docker-image tag(s) for pre-release git-tags
+        if: ( startsWith(github.ref, 'refs/tags/') && contains(github.ref, '-') )
+        run: |
+          # ie: typo3solr/ext-solr:11.5.x-dev
+          TAGS="${DOCKER_IMAGE_NAME}:${GITHUB_REF_NAME%.*}.x-dev"
+
+          echo "DOCKER_IMAGE_TAGS=${TAGS}" | tee -a $GITHUB_ENV
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      -
+        name: Download solrci-image from "ci_bootstrapping" job
+        uses: actions/download-artifact@v3
+        with:
+          name: solrci-image
+          path: /tmp
+      -
+        name: Load image
+        run: |
+          docker load --input /tmp/solrci-image.tar
+          docker image ls -a
+      -
+        name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Build and push
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          file: ./Docker/SolrServer/Dockerfile
+          platforms: ${{ env.DOCKER_IMAGE_TAGS }}
```

Comment:
Here is a tiny glitch - should be `${{ env.DOCKER_PLATFORMS }}` - i'll fix it ASAP.